lib/matplotlib/tests/test_fontconfig_pattern.py

```diff
@@ -70,8 +69,3 @@ def test_fontconfig_str():
                            stretch="expanded")
     for k in keys:
         assert getattr(font, k)() == getattr(right, k)(), test + k
-
-
-def test_fontconfig_unknown_constant():
-    with pytest.warns(DeprecationWarning):
```

Comment:
Instead of deleting this test, could you change the warning check to an error check? That way we can make sure the error is raised as expected.

Reply:
For sure, I'll do that!

Reply:
I've added the test, please check